libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -902,12 +903,38 @@ - (SFSDKUserAgentCreationBlock)defaultUserAgentString {
                                  appTypeStr,
                                  (qualifier != nil ? qualifier : @""),
                                  uid,
-                                 [[[SFSDKAppFeatureMarkers appFeatures].allObjects sortedArrayUsingSelector:@selector(caseInsensitiveCompare:)] componentsJoinedByString:@"."]
+                                 [[[SFSDKAppFeatureMarkers appFeatures].allObjects sortedArrayUsingSelector:@selector(caseInsensitiveCompare:)] componentsJoinedByString:@"."],
```

Comment:
@bjpang You need to add another space separated `%@` for the new field in line 896 for this to work. Also remove the trailing comma after `webViewUserAgent`.

Reply:
You can push a commit with the changes - you don't have to overwrite commit history.

Reply:
Ooh nice catch. I've updated it now.